Add Spanish language support for slash commands

MiguVT opened the pull request:
- Created `es-419.json` for Latin American Spanish translations.
- Created `es-ES.json` for European Spanish translations.
- Translated various music bot slash commands and responses to enhance user experience for Spanish-speaking users.

Indent was set automatically with an extension I use (Prettier)

MiguVT committed:
Add Spanish language support for music bot commands

- Created `es-419.json` for Latin American Spanish translations.
- Created `es-ES.json` for European Spanish translations.
- Translated various music bot commands and responses to enhance user experience for Spanish-speaking users.

MiguVT commented:
This should close https://github.com/ChocoMeow/Vocard/issues/74

MiguVT committed:
Merge branch 'beta' into main

MiguVT commented:
Tried to resolve conflicts using github and its really harder than VSCode... Will update it now using VSCode

MiguVT committed:
Add silent messaging support to language files for Spanish and fixed zh-TW conflict

MiguVT commented:
done :)

ChocoMeow commented:
I'm not sure if you've tried resyncing the bot to load the localization translations, but I attempted it and found that most of them don't match the validation regex for Discord string values. Here is the information of [How to resync the bot](https://docs.vocard.xyz/2.7.1/bot/config/language/#how-to-re-sync-the-bot)

ChocoMeow committed:
Merge branch 'beta' into pr/75

MiguVT commented:
Will look to it when I have time <3

MiguVT committed:
Merge branch 'beta' into main

MiguVT committed:
Updates translations for Spanish

Updates the Spanish translations in both es-419 and es-ES language files.

This commit improves the accuracy and consistency of the translations, focusing on command names and descriptions to provide a better user experience.

MiguVT commented:
Found that the problem is with discord API will implement a fix :3

MiguVT committed:
Improves command synchronization handling

Adds a smart command sync strategy to handle potential command conflicts during bot startup.

This implementation includes:
- Initial attempt at a normal sync.
- Conflict resolution mechanism via Discord's bulk update endpoint if "Entry Point command" errors are encountered.
- Error handling and logging for increased reliability.

MiguVT commented:
Fixed and everything works well also not only made it better i made logging better for command sync :)

ChocoMeow commented:
I believe the command shouldn’t trigger a sync every time the bot starts. We have a method to resync the bot as outlined in the documentation: [Resyncing the Bot](https://docs.vocard.xyz/2.7.1/bot/config/language/#how-to-re-sync-the-bot).

ChocoMeow committed:
Add new keys to Spanish language files

Added 'Fairqueue' and 'togglecontroller' keys to both es-419.json and es-ES.json for improved localization support. No other changes were made to the translation content.

MiguVT commented:
Oh ty, mb, srry for the bad quality PR i was really doing all fast cause I have too much projects 🥲
